spec/features/user/user_creation_sends_email_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.